util/src/main/java/tc/oc/pgm/util/xml/XMLUtils.java

```diff
@@ -482,17 +482,22 @@ public static Boolean parseBoolean(@Nullable Attribute attr, Boolean def)
     }
   }
 
-  public static Duration parseDuration(Node node, Duration def) throws InvalidXMLException {
+  public static Duration parseDuration(Node node, Duration def, Range<Duration> validRange)
+      throws InvalidXMLException {
     if (node == null) {
       return def;
     }
     try {
-      return TextParser.parseDuration(node.getValueNormalize());
+      return TextParser.parseDuration(node.getValueNormalize(), validRange);
     } catch (TextException e) {
-      throw new InvalidXMLException("invalid time format", node, e);
+      throw new InvalidXMLException("invalid time format: " + e.getLocalizedMessage(), node);
```

Comment:
`getLocalizedMessage` will return the `en_US` translation. If you pass the exception through, I believe that formater will properly translate it.